### PR TITLE
ui(fix): supported ellipses in tags and remove old componet

### PR DESCRIPTION
**Pull request by Ashish8689**
### Describe your changes:

- Removed old tag component
- supported ellipses in tags
- minor UI fixes around label in data header

image
image

#
### Type of change:

- [ ] Bug fix
- [ ] Improvement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/components/Tag/TagsV1/TagsV1.component.tsx [hunk 1]
```diff
@@ -88,11 +93,12 @@ const TagsV1 = ({ tag, startWith, showOnlyName = false }: TagsV1Props) => {
 
   const tagContent = useMemo(
     () => (
-      <div className="d-flex">
+      <div className="d-flex w-full">
         {tagColorBar}
-        <span className="d-flex items-center p-x-xs">
+        <span className="d-flex items-center p-x-xs w-full">
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/Tag/TagsV1/TagsV1.component.tsx:98]: If this element is a container please use div instead span

File: openmetadata-ui/src/main/resources/ui/src/styles/app.less [hunk 2]
```diff
@@ -476,3 +476,21 @@ a[href].link-text-grey,
   height: @entity-details-tab-height;
   overflow-y: scroll;
 }
+
+/* Diff style */
+
+.diff-added {
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/styles/app.less:482]: @karanh37 do you remember why we haven't add this globally?

Comment (karanh37) [reply openmetadata-ui/src/main/resources/ui/src/styles/app.less:482]: It was there globally but was part of temp.css(which we are going to remove).
I asked @Ashish8689 to move it to app.less and use variables.

**Comment by Ashish8689** [comment]
@chirag-madlani  changes made as per comments!

**Review by chirag-madlani — APPROVED** [review]

**Comment by chirag-madlani** [comment]
Cypress is failing can you check @Ashish8689

**Review by chirag-madlani — APPROVED** [review]

**Pull request merged**